test/test_alert_manager.cpp

```diff
@@ -241,7 +241,7 @@ TORRENT_TEST(wait_for_alert)
 	alert* a = mgr.wait_for_alert(seconds(1));
 
 	time_point end = clock_type::now();
-	TEST_EQUAL(a, NULL);
+	TEST_EQUAL(a, static_cast<alert*>(0));
```

Comment:
Without reverting I've only got the one from test_bencoding at hand, which is similar:
```
In file included from /home/jpetso/dev/support/libtorrent-rasterbar/test/test_bencoding.cpp:42:0:
/home/jpetso/dev/support/libtorrent-rasterbar/test/test_bencoding.cpp: In function ‘void unit_test_bencoding()’:
/home/jpetso/dev/support/libtorrent-rasterbar/test/test.hpp:121:69: warning: passing NULL to non-pointer argument 1 of ‘std::basic_ostream<_CharT, _Traits>::__ostream_type& std::basic_ostream<_CharT, _Traits>::operator<<(long int) [with _CharT = char; _Traits = std::char_traits<char>; std::basic_ostream<_CharT, _Traits>::__ostream_type = std::basic_ostream<char>]’ [-Wconversion-null]
    s__ << "TEST_EQUAL_ERROR: " #x ": " << (x) << " expected: " << (y); \
                                                                     ^
/home/jpetso/dev/support/libtorrent-rasterbar/test/test_bencoding.cpp:474:3: note: in expansion of macro ‘TEST_EQUAL’
   TEST_EQUAL(ps.ptr, NULL);
   ^
```
As far as I can tell, it regards NULL as long int and real pointer types get a different ostream overload that doesn't result in a warning. I guess using `nullptr` will be fine once libtorrent switches to C++11, this seemed like the second best phrasing given that `nullptr` does't yet exist here.